corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -23,6 +26,41 @@
 from dimagi.utils.logging import notify_exception
 
 
+class DirtinessFlag(object):
```

Comment:
FWIW, this would be a good candidate for `namedtuple`: `DirtinessFlag  = namedtuple('DirtinessFlag', ['case_id', 'owner_id'])` here—essentially equivalent, but it's a more strict data structure, and makes it clear idiomatically that you're just declaring a tiny helper data structure.